test/corpus/expressions.txt

```diff
@@ -343,6 +343,7 @@ class Foo {
   static one(a) { return a; };
   two(b) { return b; }
   three(c) { return c; }
+  #four(d) { return d; }
```

Comment:
It doesn't look like there is a test for a private property in a *member expression* (which I see you added support for in the grammar). Is that indeed allowed?

If so, maybe you could update this line to say `{ return this.#d }`, or something like that.